spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-cloudfoundry/src/main/resources/application.yml

```diff
@@ -0,0 +1,20 @@
+---
+cloudfoundry:
```

Comment:
This will typically not be read once this code is inside a bigger project at runtime (only one `application.yml` file is read on the classpath, the first one that is found, which is non-deterministic).
Defaults, *if we want them* should be set otherwise (_e.g._ as field initializers)